css-flexbox/justify-content.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - justify-content</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      #card-deck {
+        justify-content: center;
+        display: flex;
+      }
+      .card-wrapper {
+        flex-basis: 25%;
+        justify-content: center;
```

Comment:
`justify-content` is a parent property, which means it gets applied where you also apply `display: flex`. So this isn't doing anything, so it should be removed

Reply:
![Screenshot 2023-07-22 at 11 07 49 AM](https://github.com/julieluangpraxay/c0723-code-solutions/assets/130415142/d042537b-d84e-44d7-b2bf-8d39495d8d79)